MediaBrowser.Api/Syncplay/SyncplayService.cs

```diff
@@ -207,7 +216,23 @@ public void Post(SyncplayLeaveGroup request)
         public List<GroupInfoView> Post(SyncplayListGroups request)
         {
             var currentSession = GetSession(_sessionContext);
-            return _syncplayManager.ListGroups(currentSession);
+            var filterItemId = Guid.Empty;
+            if (!String.IsNullOrEmpty(request.FilterItemId))
+            {
+                try
+                {
+                    filterItemId = Guid.Parse(request.FilterItemId);
+                }
+                catch (ArgumentNullException)
+                {
+                    Logger.LogError("ListGroups: null value for FilterItemId. Ignoring filter.");
```

Comment:
is this an error or a warning? same about below

Reply:
Maybe a warning makes more sense, because the request is not rejected.